R/calcFeDemandIndustry.R

```diff
@@ -964,7 +964,7 @@ calcFeDemandIndustry <- function(use_ODYM_RECC = FALSE) {
         industry_subsectors_en %>%
           filter(2010 == .data$year),
 
-        readSource(type = "IEA_WEO_2021", subtype = "region",
```

Comment:
not worried about backwards compatibility? But you certainly much more than I do about the whole process.

Reply:
Don't see any issues with backwards compatibility.
- `madrat::getDependencies("readIEA_WEO_2021", direction = "dout")` tells you, this function is only used here and in the corresponding calc function
- running older versions of mrremind won't crash, because the original folder structure is left on the cluster:
`/p/projects/rd3mod/inputdata/sources/IEA_WEO_2021